libraries/ArduinoOTA/ArduinoOTA.cpp

```diff
@@ -146,26 +146,26 @@ int ArduinoOTAClass::parseInt(){
   uint8_t index;
   char value;
   while(_udp_ota->peek() == ' ') _udp_ota->read();
-  for(index = 0; index < sizeof(data); ++index){
+  for(index = 0; index < sizeof(data); index++){
     value = _udp_ota->peek();
     if(value < '0' || value > '9'){
       data[index++] = '\0';
```

Comment:
For me, it would be even clearer to not incrementieren the index here, something like this:
`data[index+1] = '\0';`

Reply:
This should be data[index] = '\0';